io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,42 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=~/Applications
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=io.github.prateekmedia.appimagepool
```

Comment:
This is wrong, your application `--own-name=ID` by default. There's no need to specify it.